### PR TITLE
ci: use riot for black and flake8

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**

### CODE REVIEW

File: riotfile.py [hunk 1]
```diff
@@ -3,12 +3,41 @@
...
+            Case(
+                pys=[3.8],
+                pkgs=[
+                    ("flake8", [">=3.8,<3.9"]),
+                    ("flake8-blind-except", [""]),
+                    ("flake8-builtins", [""]),
+                    ("flake8-docstrings", [""]),
+                    ("flake8-logging-format", [""]),
+                    ("flake8-rst-docstrings", [""]),
+                    ("pygments", [""]),
```
Comment (brettlangdon) [inline riotfile.py:36]:
```suggestion
                    ("flake8", [">=3.8,<3.9"]),
                    ("flake8-blind-except", ),
                    ("flake8-builtins", ),
                    ("flake8-docstrings", ),
                    ("flake8-logging-format", ),
                    ("flake8-rst-docstrings", ),
                    ("pygments", ),
```

is this more legible?

this comment has nothing to do with this PR

File: tox.ini [hunk 2]
```diff
@@ -583,24 +581,6 @@ deps=
 # this is somewhat flaky (can fail and still be up) so try the tests anyway
 ignore_outcome=true
 
-[testenv:black]
-skip_install=true
-deps=black==20.8b1
-commands=black --check .
-
-[testenv:flake8]
-skip_install=true
```
Comment (brettlangdon) [inline tox.ini:592]: this doesn't hurt much on runtime right?

**Review by brettlangdon — APPROVED** [review]

File: riotfile.py (hunk 1 above)
Comment (Kyle-Verhoog) [reply riotfile.py:36]: We could also just have it be `[("flake8", ">=3.8,<3.9"), "flake8-blind-except", "flake8-builtins", "flake8-docstrings"]`.

Comment (brettlangdon) [reply riotfile.py:36]: yeah

File: tox.ini (hunk 2 above)
Comment (brettlangdon) [reply tox.ini:592]: doesn't seem like the runs from this PR are different from `master`

**Pull request merged**